contrib/dev/Vagrantfile

```diff
@@ -107,19 +128,47 @@ def populate_os_config(os, ip)
       address: '192.168.10.10',
     },
   ]
+  os.security_groups = [ 'skydive' ]
+end
+
+def fedora_box(override, type)
+  major = "28" 
+  minor = "1.1"
+  override.vm.box = "fedora/#{major}-cloud-base"
+  override.vm.box_url = "http://mirror.isoc.org.il/pub/fedora/releases/#{major}/Cloud/x86_64/images/Fedora-Cloud-Base-Vagrant-#{major}-#{minor}.x86_64.vagrant-#{type}.box"
```

Comment:
I would prefer to use download.fedoraproject.org in case this one is down

Reply:
Can you give me a full URL? - I can't find the cloud image for fc28 here